pybaseball/split_stats.py

```diff
@@ -0,0 +1,116 @@
+import requests
+from requests.adapters import HTTPAdapter
+from requests.packages.urllib3.util.retry import Retry
+import bs4 as bs
+import pandas as pd
+import re
+
+def download_url(url):
```

Comment:
This looks really interesting, and would like to see this in a place like `datahelpers/http.py` so others could use it as well.

Reply:
Are you saying to create a new file at datahelpers/http.py? Or am I just not finding it?

Reply:
How is this vs `requests.get`? Performance differences? Reasons to prefer it?

Reply:
I would put this in `utils.py`. Although I would like to hear out the benefits of this vs the unmodified `requests.get()`

Reply:
> How is this vs `requests.get`? Performance differences? Reasons to prefer it?

I'll let @wisnie5 comment on why they thought this would be useful, but the immediate utility I see are auto retries on failures.

Reply:
I actually built this function for another project, but it was to avoid errors when the website wasn't responding. I found that when I called sites a ton of times in a row, I would have requests get error responses. This function gives it a few more tries before failing and I am no longer seeing the no response errors. .